README_IT.md

```diff
@@ -64,6 +64,7 @@ Da usare con la Carpet Mod nella stessa versione di Minecraft. È suggerito usar
 - [stonecutterWood](#stonecutterWood)
 - [treecapitator](#treecapitator)
 - [villagerLeash](#villagerLeash)
+- [playerme](#playerme)
```

Comment:
Da posizionare nel punto corretto, al fine che le regole siano disposte in ordine alfabetico

Reply:
da rinominare in commandPlayerme per consistenza